src/backend/distributed/sql/downgrades/citus--12.0-1--11.3-1.sql

```diff
@@ -30,6 +30,9 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+DROP FUNCTION IF EXISTS pg_catalog.citus_schema_distribute(regnamespace);
+DROP FUNCTION IF EXISTS pg_catalog.citus_schema_undistribute(regnamespace);
```

Comment:
do we really need "IF EXISTS"?

Reply:
I always use if exists while dropping if syntax allows. I think idempotent commands are better to not fail during migration scripts. Do we have any disadvantage of using it?

Reply:
Both are fine to me. However, apparently we didn't prefer IF EXISTS syntax much in the latest releases of Citus (e.g.: downgrades/citus--11.1-1--11.0-4.sql), so I'd vote for not using IF EXISTS syntax here.